modules/monitoring-getting-information-about-alerts-silences-and-alerting-rules.adoc

```diff
@@ -10,87 +10,79 @@ The Alerting UI provides detailed information about alerts and their governing a
-+
---
-** A description of the alert
-** Messages associated with the alerts
-** Labels attached to the alert
-** A link to its governing alerting rule
-** Silences for the alert, if any exist
---
+. Click the name of an alert to view its *Alert details* page. The page includes a graph that illustrates alert time series data. It also provides the following information about the alert:
+
+* A description of the alert
+* Messages associated with the alert
+* Labels attached to the alert
+* A link to its governing alerting rule
+* Silences for the alert, if any exist
 
 *To obtain information about silences in the Administrator perspective*:
 
-. Navigate to the *Observe* -> *Alerting* -> *Silences* page.
+. Go to the *Observe* -> *Alerting* -> *Silences* page.
 
 . Optional: Filter the silences by name using the *Search by name* field.
 
 . Optional: Filter silences by state by selecting filters in the *Filter* list. By default, *Active* and *Pending* filters are applied.
 
-. Optional: Sort the silences by clicking one or more of the *Name*, *Firing Alerts*, and *State* column headers.
+. Optional: Sort the silences by clicking one or more of the *Name*, *Firing alerts*, *State*, and *Creator* column headers.
+
+. Select the name of a silence to view its *Silence details* page. The page includes the following details:
 
-. Select the name of a silence to navigate to its *Silence Details* page. The page includes the following details:
-+
---
 * Alert specification
 * Start time
 * End time
 * Silence state
 * Number and list of firing alerts
---
 
 *To obtain information about alerting rules in the Administrator perspective*:
 
-. Navigate to the *Observe* -> *Alerting* -> *Alerting Rules* page.
+. Go to the *Observe* -> *Alerting* -> *Alerting rules* page.
 
 . Optional: Filter alerting rules by state, severity, and source by selecting filters in the *Filter* list.
 
-. Optional: Sort the alerting rules by clicking one or more of the *Name*, *Severity*, *Alert State*, and *Source* column headers.
+. Optional: Sort the alerting rules by clicking one or more of the *Name*, *Severity*, *Alert state*, and *Source* column headers.
+
+. Select the name of an alerting rule to view its *Alerting rule details* page. The page provides the following details about the alerting rule:
 
-. Select the name of an alerting rule to navigate to its *Alerting Rule Details* page. The page provides the following details about the alerting rule:
-+
---
-** Alerting rule name, severity, and description
-** The expression that defines the condition for firing the alert
-** The time for which the condition should be true for an alert to fire
-** A graph for each alert governed by the alerting rule, showing the value with which the alert is firing
-** A table of all alerts governed by the alerting rule
---
+* Alerting rule name, severity, and description
+* The expression that defines the condition for firing the alert
```

Comment:
Consider a full stop for each listed item as each is a full sentence.

Reply:
Thank you for catching that one!